packages/dapp-svelte-wallet/ui/src/Transaction.svelte

```diff
@@ -25,7 +25,6 @@
     if (typeof outcome !== 'string') {
```

Comment:
This whole function should go away.  If you just want to silence a dapp in the meantime, use an offer where `offer.dappContext = true`.